framework/src/play/Play.java

```diff
@@ -158,6 +153,13 @@ public void init(String id) {
     setupApplicationMode();
     VirtualFile appRoot = setupAppRoot();
     routes = appRoot.child("conf/routes");
+    if (!routes.exists()) {
+      try {
+        routes = VirtualFile.open(Objects.requireNonNull(Thread.currentThread().getContextClassLoader().getResource("conf/routes")).getFile());
```

Comment:
Which is better?
* `.getContextLoader().getResource()` + relative path: `conf/...`, or
* `getClass().getResource` + absolute path: `/conf/...`?

Both works. :woman_shrugging:

Reply:
I am not sure, but it seems to me reasonable to use `getContextLoader()` because it gives users to use some custom classloader (for whatever reason).